plugins/generator-1.19.4/forge-1.19.4/templates/elementinits/items.java.ftl

```diff
@@ -68,8 +68,8 @@ public class ${JavaModName}Items {
 		<#elseif item.getModElement().getTypeString() == "fluid" && item.generateBucket>
 			public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()}_BUCKET =
 				REGISTRY.register("${item.getModElement().getRegistryName()}_bucket", () -> new ${item.getModElement().getName()}Item());
-		<#elseif item.getModElement().getTypeString() == "dimension">
-			public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()} =
+		<#elseif item.getModElement().getTypeString() == "dimension" && item.enableIgniter>
+            public static final RegistryObject<Item> ${item.getModElement().getRegistryNameUpper()} =

```

Comment:
This line isn't indented properly (same in 1.19.2)

Reply:
I don't see this issue in 1.19.2

Reply:
Anyways, issue fixed

Reply:
The 1.19.2 template also uses spaces instead of tabs in the same place

Reply:
No it doesn't there is only 1 changed line in 1.19.2 template (excepted the year)

Reply:
The fact that it isn't changed is because it used the spaces before as well. You could fix that in this PR